tests/test_traceback.py

```diff
@@ -263,6 +284,36 @@ def __str__(self):
     assert "<exception str() failed>" in result
 
 
+def test_broken_rich_exception():
+    class BrokenRichError(Exception):
+        def __rich_console__(self, console, options):
+            raise Exception("broken")
+
+    console = Console(width=100, file=io.StringIO())
+    try:
+        raise BrokenRichError()
+    except Exception:
+        console.print_exception()
+    result = console.file.getvalue()
+    print(result)
```

Comment:
These prints are useful for diagnosing failures, and pytest will capture this output appropriately. :)